lib/ardb.rb

```diff
@@ -27,6 +27,9 @@ def self.init(establish_connection = true)
     # setup AR
     ActiveRecord::Base.logger = self.config.logger
     self.adapter.connect_db if establish_connection
+  rescue InvalidDBFileError => exception
+    exception.set_backtrace(caller)
+    raise exception
```

Comment:
@kellyredding - Yeah, I was altering the backtrace for non-cli usage, good call on the sweeping rescue.